### ISSUE TITLE
Solarized light color scheme renders some of sprintly's output unreadable

### ISSUE THREAD

**Issue by kvikshaug** [sprintly/Sprintly-GitHub#34]
I'm using the [Solarized color scheme](http://ethanschoonover.com/solarized) and as you can see, Sprintly's chosen colors renders the item number for task items (second line in backlog) unreadable:

![Screenshot](http://i.imgur.com/NzhUETG.png)

When i copy the text and paste it in my editor, it shows that the item number is there - just unreadable. Is this an issue with how Sprintly chooses colors, or a weakness of Solarized's light color scheme?

**Comment by justinabrahms** [comment]
https://github.com/sprintly/Sprintly-GitHub/blob/master/sprintly#L39 is where we calculate colors and those numbers are colored based on `ITEM_COLORS` a few lines down. Search for `ITEM_COLORS` to see where that gets used.

It looks like the number for defects is showing just fine. Is it maybe a task and the `GREY` color isn't showing?

**Comment by kvikshaug** [comment]
That's right - it's only for tasks, in other words only `GREY`, which don't appear, here's a more clear example:

![screenshot](http://i.imgur.com/LG94xD3.png)

I'm not well versed with how these terminal colors work, so again, I can't pinpoint whether it's sprintly or just the color scheme that's in the wrong here - just thought I should give a heads up.

**Comment by justinabrahms** [comment]
If you'd like to take a swing at fixing this, maybe try some other colors
in that type mapping until your issue is fixed. :)
